ml.net/InclusiveCodeReviews.Model/ModelInput.cs

```diff
@@ -17,6 +17,9 @@ public class ModelInput
         [ColumnName("isnegative"), LoadColumn(1)]
         public string Isnegative { get; set; }
 
+        [ColumnName("importance"), LoadColumn(2)]
+        public float Importance { get; set; }
```

Comment:
Should this be:
```suggestion
        [ColumnName("importance"), LoadColumn(2)]
        public float Importance { get; set; } = 0.5f;
```
Then we might test the MLTrainer app can add a row? And it just always puts 0.5 by default?